### PR TITLE
Show short and long variable names in message

### PR DESCRIPTION
**Pull request by vinyldarkscratch**

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
Oof, not liking the looks of the coverage drop...

File: oclint-rules/test/naming/LongVariableNameRuleTest.cpp [hunk 1]
```diff
@@ -46,11 +46,11 @@ TEST_F(LongVariableNameRuleTest, ThreeCharsName)
 TEST_F(LongVariableNameRuleTest, FourCharsName)
 {
     testRuleOnCode(new LongVariableNameRule(), "void aMethod(int iiii) {}",
-        0, 1, 14, 1, 18, "Variable name with 4 characters is longer than the threshold of 3");
+        0, 1, 14, 1, 18, "Variable name with 4 characters (iiii) is longer than the threshold of 3");
```
Comment (ryuichis) [inline oclint-rules/test/naming/LongVariableNameRuleTest.cpp]: Do you think a pair of backtick "`" might be better than parenthesis? 

And I am also thinking to make it "Length of variable name `iiii` is 4, which is longer than threshold of 3".

I am adding the comment here, but it should apply to all changes.

Comment (vinyldarkscratch) [inline oclint-rules/test/naming/LongVariableNameRuleTest.cpp]: Now that you mention it, backticks (or other types of quotes) would definitely be better, I like that, yeah!  Applying changes very shortly.  ;)

**Comment by ryuichis** [comment]
Thanks for working on this @vinyldarkscratch

**Comment by vinyldarkscratch** [comment]
Of course, @ryuichis!  Glad to help out however I can.  ;)

**Pull request merged**